src/main/java/io/cryostat/net/reports/AbstractReportGeneratorService.java

```diff
@@ -85,8 +85,7 @@ Path copyRecordingToFile(
                 continue;
             }
             try (OutputStream out = new BufferedOutputStream(new FileOutputStream(path.toFile()))) {
-                try (conn;
-                        InputStream in = conn.getService().openStream(rec, false)) {
+                try (InputStream in = conn.getService().openStream(rec, false)) {
```

Comment:
What was the reason for this change? It doesn't seem related to the rest of the patch.

Reply:
it was just because spotbug was failing ...

Reply:
Weird - it doesn't seem to normally fail here. I wonder what made it pick this up as a bug. Was it complaining about the `conn` being a null pointer? I've seen that one before, it's some false positive that happens when you use `try-with-resources` blocks.

Speaking of which though, this nesting looks silly anyway:

```java
            try (OutputStream out = new BufferedOutputStream(new FileOutputStream(path.toFile()))) {
                try (conn;
                        InputStream in = conn.getService().openStream(rec, false)) {
                    byte[] buff = new byte[READ_BUFFER_SIZE];
                    int n = 0;
                    while ((n = in.read(buff)) != -1) {
                        out.write(buff, 0, n);
                        if (!targetConnectionManager.markConnectionInUse(cd)) {
                            throw new IOException(
                                    "Target connection unexpectedly closed while streaming"
                                            + " recording");
                        }
                    }
                    out.flush();
                    return path;
                }
            }
```

Looks like those two `try` blocks can be combined into one.

Reply:
yea that's what failed it. I'll add it back in and merge the try blocks